index.bs

```diff
@@ -128,6 +129,8 @@ Physical Hand Input Sources {#physical-hand}
 
 An {{XRInputSource}} is a <dfn>physical hand input source</dfn> if it tracks a physical hand. A [=physical hand input sources=] <dfn>supports hand tracking</dfn> if it supports reporting the poses of one or more [=skeleton joint=]s defined in this specification.
 
+[=Physical hand input sources=] SHOULD include the [=XRInputSource/input profile name=] of "generic-hand-select" in their {{XRInputSource/profile}}.
```

Comment:
can this be a MUST?

Reply:
It could, I was being cautious for now. I'll change it.